src/pages/Content/utils/element.ts

```diff
@@ -0,0 +1,11 @@
+function arrayFrom(
+  collectionOrNodeList: NodeList | HTMLCollectionOf<Element>
+): HTMLElement[] {
+  return Array.prototype.slice.call(collectionOrNodeList);
+}
```

Comment:
collectionOrNodeList 라는 네이밍보다는 제네릭을 사용해주면 어떨까요?
```suggestion
function arrayFrom<Elements extends NodeList | HTMLCollectionOf<Element>>(
  elements: Elements
): HTMLElement[] {
  return Array.prototype.slice.call(elements);
}
```

Reply:
https://github.com/parkhojeong/commits-for-github/pull/3/commits/bca7970ee9a1cf558bf6840fa9ea95ff4d4e466a

요 유틸함수는 위 커밋에서 지워버렸습니당